devel/example_notebooks/example1.ipynb

```diff
@@ -60,7 +60,7 @@
    "outputs": [],
    "source": [
     "# Now retrieve the curation object here\n",
-    "curation = X.curation\n",
+    "curation = X.get_curation()\n",
```

Comment:
Replace `X.curation` by `X.get_curation()` and `X.set_curation(c)`. This was needed to create deep copies of the curation object, which appeared to be needed to trigger the sync. Similar for selection and external_unit_metrics